components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +159,556 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
+
+*In the browser:* **Settings**  >> **Teams** >> [select a team] >> **Users** >> [open control menu for target user] >> Remove User
+
 ## Tokens
 
-Tokens are unchanged for IAM v2.
-To specify tokens in a [member expression]({{< relref "iam-v2-overview.md#members-and-policies" >}}), first find that token's ID.
+Method | HTTP request              | Description
+-------|---------------------------|------------
+list   | GET /tokens               | lists all tokens
+get    | GET /tokens/***id***      | gets the specified token
+create | POST /tokens              | creates a token
+update | PUT /tokens/***id***      | updates a token
+delete | DELETE /tokens/***id***   | deletes a token
+
+**Example Token:**
+
+```json
+{
+  "name": "token 1",
+  "id": "token-1",
+  "active": true,
+  "projects": [
+    "east-region",
+    "west-region"
+  ]
+}
+```
 
 ### Listing Tokens
 
-List all tokens with this command:
+The lists admin and non-admin tokens.
+
+*In the browser:* **Settings**  >> **API Tokens**
+
+### Getting a Token
+
+This shows the details of a single token, selected by its ID.
+
+*In the browser:* **Settings**  >> **API Tokens** >> [select a token]
+
+### Creating a Token
+
+Create a token by composing JSON with all the necessary token properties (see the beginning of the section above).
+Note that this creates **non**-admin tokens that may then be assigned permissions via policies just like users or teams (unless you have already created policies that encompass all tokens using `tokens:*`).
+
+You cannot create admin tokens via the REST API.
+Admin tokens can only be created by specifying the `--admin` flag to this `chef-automate` sub-command:
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/api/v0/auth/tokens?pretty
+  chef-automate iam token create <your-token-name> --admin
 ```
 
+*In the browser:* **Settings**  >> **API Tokens** >> Create Token
+
+### Updating a Token
+
+The update operation modifies *all* token properties, not just the ones you specify.
+Properties that you do not specify are reset to empty values.
+Thus, you should supply all of a token's properties, not just the ones you wish to update.
+Note that the ID is immutable; it can only be set at creation time.
+
+*In the browser:* **Settings**  >> **API Tokens** >> [select a token] >> Details
+
+### Deleting a Token
+
+Deleting a token is permanent and cannot be undone.
+
+*In the browser:* **Settings**  >> **API Tokens** >> [open control menu for target token] >> Delete Token
+
 ## Restoring Admin Access
 
 While we have [safeguards]({{< relref "iam-v2-overview.md#policy-types" >}}) to prevent it, it is possible to lock yourself out of Chef Automate.
 If you have root access to the node where Chef Automate is installed, use the following commands to restore admin access:
 
 <!-- Wording/Replace password/replace token/v1 commands won't work on c2 -->
```

Comment:
No idea what this comment is trying to say. If someone does, say so, otherwise would like to delete it.